### ISSUE TITLE
Build error after pod install

### ISSUE THREAD

**Issue by raykle** [vonovak/react-native-simple-toast#15]
![image](https://user-images.githubusercontent.com/3097366/118933033-4edaa500-b97b-11eb-8e0b-5951a5918743.png)

Any help?

<details>
  <summary>react-native info</summary>

  ```
  System:
    OS: macOS 11.2.3
    CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
    Memory: 22.58 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 12.22.0 - ~/.nvm/versions/node/v12.22.0/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v12.22.0/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.10.1 - /Users/guomin/.rvm/gems/ruby-2.6.6/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 23, 25, 28
      Build Tools: 28.0.3, 29.0.0
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 12.4/12D4e - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_101 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1 
    react-native: 0.64.0 => 0.64.0 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
  ```
</details>

**Comment by adamgajzlerowicz** [comment]
This is like the most generic react-native error. Are you still experiencing it?

**Comment by SamiAljord** [comment]
> This is like the most generic react-native error. Are you still experiencing it?

I still do, any recommendations?

**Comment by vonovak** [comment]
should be fixed in 1.1.4 :)

**Comment by adamgajzlerowicz** [comment]
Busy weekend? 😄
